features/yield/FarmingCalculator.tsx

```diff
@@ -259,13 +293,15 @@ const FarmingCalculator = () => {
         <br></br>
         <Typography>
           <strong>Pools eligible for liquidity mining:</strong>{" "}
-          {isRolled ? "yUSD-OCT, uUSDrBTC-OCT" : "SEPT + OCT"}
+          {timeUntilRoll
+            ? isRolled
+              ? "NOV yield tokens"
```

Comment:
I would also perhaps want these dates to be stored within `ROLL_REWARDS_SCHEDULE`

Reply:
I can just read them from the ROLL_REWARDS_SCHEDULE start and end dates. Will fix

Reply:
I now read the token symbol dynamically and display either the current pool or the current pool + roll pool